flop

```diff
@@ -137,6 +144,13 @@ EOF
                         mkdir -p ../modules/stac-server/lambda/pre-hook
                         cp dist/pre-hook/pre-hook.zip ../modules/stac-server/lambda/pre-hook/
                         cd ..
+                        cd modules/stac-server/historical-ingest/lambda/
```

Comment:
This means that even if `include_historical_ingest = false` it will still package up the lambda, right? Do we want to make this optional and only if `include_historical_ingest  = true`?

Do we need a similar approach for the ongoing ingest? I don't see that here.

Reply:
We should talk about this one – if you're doing LYOSTAC and choose to exclude historical ingest (yes ongoing, no historical), you actually still need this module because it's what creates the Collections.

But as for building / deploying it in all cases not-LYOSTAC, that's a fair question

Reply:
I just modified this script to only build the stac server dependencies if `deploy_stac_server = true` and only build the historical ingest packages if `include_historical_ingest = true`.